GitUpKit/Views/Base.lproj/GIAdvancedCommitViewController.xib

```diff
@@ -131,10 +131,30 @@
                                                                         </userDefinedRuntimeAttribute>
                                                                     </userDefinedRuntimeAttributes>
                                                                 </customView>
-                                                                <customView id="SLl-qM-6U5">
-                                                                    <rect key="frame" x="0.0" y="0.0" width="300" height="168"/>
+                                                                <customView fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="SLl-qM-6U5">
+                                                                    <rect key="frame" x="0.0" y="26" width="300" height="142"/>
                                                                     <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                                                 </customView>
+                                                                <customView id="0Ii-fZ-SOT">
+                                                                    <rect key="frame" x="0.0" y="0.0" width="300" height="27"/>
+                                                                    <autoresizingMask key="autoresizingMask" widthSizable="YES"/>
+                                                                    <subviews>
+                                                                        <searchField wantsLayer="YES" verticalHuggingPriority="750" textCompletion="NO" translatesAutoresizingMaskIntoConstraints="NO" id="u4y-Fx-A8g">
```

Comment:
The search field should have placeholder text "Filter". We're not really searching any of the files.